### PR TITLE
feat : added authentication pages

### PR DESCRIPTION
**Pull request by sharmivijay**
Kindly review and add your comments

### CODE REVIEW

File: styles/login.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+body{
+    background-color: rgb(217, 243, 166);
```
Comment (harshalini) [inline styles/login.css]: Try to use variables for color, rest everything is good

File: styles/signup.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
+body{
+    background-color: rgb(217, 243, 166);
```
Comment (harshalini) [inline styles/signup.css]: Here also same as above

**Review by harshalini — APPROVED** [review]

**Review by raghav2110 — APPROVED** [review]
Overall UI can be enhanced a bit. Code looks clean and naming of CSS classes are also good. Just make the changes as suggested.

File: styles/login.css (hunk 1 above)
Comment (sharmivijay) [reply styles/login.css]: sure. i will add the variables files

**Pull request merged**